shared/profile/index.native.js

```diff
@@ -418,31 +418,28 @@ class Profile extends Component<Props, State> {
     if (section.title === 'profile') {
       const trackerStateColors = stateColors(this.props.currentlyFollowing, this.props.trackerState)
       return (
-        <Kb.Box
+        <Kb.HeaderHocHeader
```

Comment:
The blue background isn't staying behind the search bar as you scroll
<img width="394" alt="fullscreen_1_14_19__10_03_am" src="https://user-images.githubusercontent.com/11968340/51120730-bd275f80-17e3-11e9-9072-23c08d40125a.png">